packages/kit/src/exports/node/index.js

```diff
@@ -109,7 +111,9 @@ export async function getRequest({ request, base, bodySizeLimit }) {
 		duplex: 'half',
 		method: request.method,
 		headers: /** @type {Record<string, string>} */ (request.headers),
-		body: get_raw_body(request, bodySizeLimit)
+		body: can_have_body.includes(request.method ?? '')
+			? get_raw_body(request, bodySizeLimit)
+			: undefined
```

Comment:
Is this faster than an `||` (since it's a fixed list)? What about a `Set`? I'm worried about adding unnecessary overhead (however small) to every single request.

Reply:
`||` will be faster, set will be much slower. fixed in #11708